lib/token.coffee

```diff
@@ -118,6 +122,7 @@ exports.has = ->
 # token.remove()
 ###
 exports.remove = ->
+	delete process.env.RESIN_TOKEN
```

Comment:
Not sure about whether removing the env var is expected. I'd say we don't remove it, but I won't insist.